CHANGELOG.mkd

```diff
@@ -5,8 +5,13 @@ Unreleased
 ----------
 ## Changes
 
-(#823) Add filter_command configuration option for git repositories
-(Thanks to [mhumpula](https://github.com/mhumpula) for the [feature](https://github.com/puppetlabs/r10k/pull/823/conflicts).)
+- (#823) Add filter_command configuration option for git repositories
+(Thanks to [mhumpula](https://github.com/mhumpula) for the [feature](https://github.com/puppetlabs/r10k/pull/823).)
+- Increase default pool_size to 4, allowing modules to be downloaded on 4 threads concurrently. [#1038](https://github.com/puppetlabs/r10k/issues/1038)
+
+## Bug Fix
```

Comment:
Looks like we haven't been categorizing our changes since 3.3.3.
I know the `## Changes` above was introduced by a community PR.. 
Do we want to go back to categorizing like this? If so, do we want to fix the last few versions or nah?

Reply:
No let's not.

Reply:
Let's not fix old versions or let's not re-start doing it this way at all?

Reply:
Not re-start doing it.